tests/functional/raw-node/test/GCP/object/deleteTagging.js

```diff
@@ -0,0 +1,148 @@
+        });
+        assert.deepStrictEqual(params.tag, tagRes);
+        assert.deepStrictEqual(params.meta, metaRes);
+        return callback();
+    });
+}
+
+describe('GCP: DELETE Object Tagging', function testSuite() {
+    this.timeout(30000);
+
+    before(done => {
+        config = getRealAwsConfig(credentialOne);
+        gcpClient = new GCP(config);
+        gcpRequestRetry({
+            method: 'PUT',
+            bucket: bucketName,
+            authCredentials: config.credentials,
+        }, 0, err => {
+            if (err) {
+                process.stdout.write(`err in creating bucket ${err}`);
+            }
+            return done(err);
+        });
+    });
+
+    beforeEach(function beforeFn(done) {
+        this.currentTest.key = `somekey-${Date.now()}`;
+        this.currentTest.specialKey = `veryspecial-${Date.now()}`;
+        const { headers, expectedTagObj, expectedMetaObj } =
+            genDelTagObj(10, gcpTagPrefix);
+        this.currentTest.expectedTagObj = expectedTagObj;
+        this.currentTest.expectedMetaObj = expectedMetaObj;
+        makeGcpRequest({
+            method: 'PUT',
+            bucket: bucketName,
+            objectKey: this.currentTest.key,
+            authCredentials: config.credentials,
+            headers,
+        }, (err, res) => {
+            if (err) {
+                process.stdout.write(`err in creating object ${err}`);
+                return done(err);
+            }
+            this.currentTest.versionId = res.headers['x-goog-generation'];
+            return done();
+        });
+    });
+
+    afterEach(function afterFn(done) {
+        makeGcpRequest({
+            method: 'DELETE',
+            bucket: bucketName,
+            objectKey: this.currentTest.key,
+            authCredentials: config.credentials,
+        }, err => {
+            if (err) {
+                process.stdout.write(`err in deleting object ${err}`);
+            }
+            return done(err);
+        });
+    });
+
+    after(done => {
+        gcpRequestRetry({
+            method: 'DELETE',
+            bucket: bucketName,
+            authCredentials: config.credentials,
+        }, 0, err => {
+            if (err) {
+                process.stdout.write(`err in deleting bucket ${err}`);
+            }
+            return done(err);
+        });
+    });
+
+    it('should successfully delete object tags', function testFn(done) {
+        async.waterfall([
+            next => assertObjectMetaTag({
+                bucket: bucketName,
+                key: this.test.key,
+                versionId: this.test.versionId,
+                meta: this.test.expectedMetaObj,
+                tag: this.test.expectedTagObj,
+            }, next),
+            next => gcpClient.deleteObjectTagging({
+                Bucket: bucketName,
+                Key: this.test.key,
+                VersionId: this.test.versionId,
+            }, err => {
+                assert.equal(err, null,
```

Comment:
lgtm, just wanted to add that you can also use `assert.ifError` in these situations